### PR TITLE
Add header and headline-section 

### PR DESCRIPTION
**Pull request by Achilles-Dev**
- [ ] Added a header to the website
- [ ] Added the headline section of the website

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>My Portfolio Website</title> 
+  <link rel="stylesheet" href="./style.css">
+</head>
+<body>
+  <header class="header">
+    <nav class="nav">
+      <div class="top-nav"><img src="./assets/icons/Light.svg" alt=""></div>
```
Comment (elmar8287) [inline index.html]: - [ ] Please, consider removing this part from your code, because you do not need to implement the mobile phone top clock, wifi, battery, etc - all these are not related to project requirements
Screen Shot 2022-02-10 at 21 57 17

File: style.css [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+  height: 6715px;
+}
+
+.header {
+  position: absolute;
+  width: 100%;
+  height: 92px;
+  left: 0;
+  top: 0;
+}
```
Comment (elmar8287) [inline style.css]: - [ ] Please, consider that the content is aligned with flexbox

File: style.css [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+  display: flex;
+  flex-direction: row;
+  align-items: flex-start;
+  padding: 8px;
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  background: #fff;
+  left: calc(50% - 40px / 2 + 0.5px);
+}
```
Comment (elmar8287) [inline style.css]: - [ ] Please, consider implementing also the Active, Hover, and Pressed (see design) for your button, as required in the project.
Screen Shot 2022-02-10 at 22 03 34

**Review by elmar8287 — CHANGES_REQUESTED** [review]
Hi @Achilles-Dev, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [ ]  Check the comments under the review.

## Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+      </p>
+      <ul class="headline-icons">
+        <li><a href="#twitter" ><img src="./assets/icons/twitter.svg" alt="Twitter"></a></li>
+        <li><a href="#linkedIn" ><img src="./assets/icons/Linkedin_icon.svg" alt="LinkedIn"></a></li>
+        <li><a href="#medium" ><img src="./assets/icons/medium.svg" alt="Medium"></a></li>
+        <li><a href="#github" ><img src="./assets/icons/github.svg" alt="Github"></a></li>
+        <li><a href="#angellist" ><img src="./assets/icons/angellist.svg" alt="Angellist"></a></li>
+      </ul>
+      <div class="headline-dropdown">
+        <a href=""><img src="/assets/icons/Disabled.svg" alt="Dropdown"></a>
```
Comment (gemmen29) [inline index.html]: [OPTIONAL] Kindly change the image's path to `"./assets/icons/Disabled.svg"` to fix the image displayed. Adding the dot means that the path starts from the current file's directory.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+        some of my  work and experience! If you like what you see and have 
+        a project you need coded, don’t hestiate to contact me.
+      </p>
+      <ul class="headline-icons">
+        <li><a href="#twitter" ><img src="./assets/icons/twitter.svg" alt="Twitter"></a></li>
+        <li><a href="#linkedIn" ><img src="./assets/icons/Linkedin_icon.svg" alt="LinkedIn"></a></li>
+        <li><a href="#medium" ><img src="./assets/icons/medium.svg" alt="Medium"></a></li>
+        <li><a href="#github" ><img src="./assets/icons/github.svg" alt="Github"></a></li>
+        <li><a href="#angellist" ><img src="./assets/icons/angellist.svg" alt="Angellist"></a></li>
+      </ul>
```
Comment (gemmen29) [inline index.html:29]: [OPTIONAL] Please center the icons to be in the middle of the screen.

File: style.css [hunk 6]
```diff
@@ -0,0 +1,148 @@
...
+  flex-direction: column;
+  align-items: center;
+  padding: 64px 24px;
+  position: absolute;
+  width: 375px;
+  height: 726px;
+  left: 0;
+  top: 92px;
+  background-image: url("assets/images/header-illsutration-mobile.png");
+}
```
Comment (gemmen29) [inline style.css]: [OPTIONAL] Kindly adjust the background image to cover the whole with of the screen and add the propert `backgorund-repeat: no-repeat` which makes the background not repeated like the Figma design and adjust it's size.

**Review by gemmen29 — APPROVED** [review]
Hi @Achilles-Dev, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions
- Please update your README file to be more like [Microverse's readme template](https://github.com/microverseinc/readme-template). Your README should be descriptive and show the project details, a screenshot, authors and contributions.
- _Check the other comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css (hunk 6 above)
Comment (Achilles-Dev) [reply style.css]: Thank you
